client/go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/kubernetes-csi/external-snapshotter/client/v6
 
-go 1.18
+go 1.19
 
 require (
-	k8s.io/api v0.26.0-rc.0
-	k8s.io/apimachinery v0.26.0-rc.0
-	k8s.io/client-go v0.26.0-rc.0
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.0
+	k8s.io/client-go v0.26.0
 	k8s.io/code-generator v0.25.2
```

Comment:
Does this one need to be 0.26.0 as well?

Reply:
This one is okay. Leave this one as is.